docs/src/faqs/setup_packaging.rst

```diff
@@ -0,0 +1,108 @@
+What do I need to know about using ``setup.py`` to package dependencies in Klio?
+================================================================================
+
+Using ``setup.py`` or a ``requirements.txt`` file to specify dependencies to install on workers was the original means by which a Python Beam job developer could customize the environment in which their job ran. You can read more about it in the `here <https://beam.apache.org/documentation/sdks/python-pipeline-dependencies/>`_.
+
+To configure a Klio job to run using ``setup.py`` packaging, the user needs to provide the required files (see below) and configure the job to use them by filling in the ``pipeline_options.setup_file`` field in their ``klio-job.yaml`` file:
+
+
+.. code-block:: yaml
+
+    job_name: my-job
+    pipeline_options:
+        setup_file: setup.py
+        # <-- snip -->
+
+Note that this method of setup is mutually exclusive with using the FnAPI for packaging.
+
+Required Files
+--------------
+
+.. _setup-py:
+
+``setup.py``
+^^^^^^^^^^^^
+
+A ``setup.py`` file is needed in the **root of your job's directory**. It partly substitutes the
+need for a worker image by installing any non-Python dependencies via a child process, and by
+explicitly including non-Python files needed for a job (i.e. a model, a JSON schema, etc).
+
+
+.. tip::
+
+    The ``setup.py`` must contain the required system-level dependencies, Python dependencies, and
+    required non-Python files (i.e. ML models, JSON schemas, etc) that your job requires to run.
+
+.. collapsible:: Minimal Example ``setup.py``
+
+    The following is an example with a non-Python file (a ML model, ``my-model.h5``), and no
+    non-public Python package dependencies or OS-level dependencies.
+
+    .. code-block:: python
+
+        import setuptools
+
+        setuptools.setup(
+            name="my-example-job",  # required
+            version="0.0.1",  # required
+            author="klio-devs",  # optional
+            author_email="hello@example.com",  # optional
+            description="My example job using setup.py",  # optional
+            install_requires=["tensorflow"],  # optional
+            data_files=[  # required
+                (".", ["klio-job.yaml", "my-model.h5"]),
+            ],
+            include_package_data=True,  # required
+            py_modules=["run", "transforms"],  # required
+        )
+
+
+.. todo:: Show example with internal & OS-level dependencies
```

Comment:
Do you want to address this todo now or later?

Reply:
Added an example!